src/parser/jobs/rdm/index.js

```diff
@@ -21,7 +21,7 @@ export default new Meta({
 	</>,
 	supportedPatches: {
 		from: '5.05',
-		to: '5.08',
+		to: '5.1',
```

Comment:
are these changes gonna work on the old patch?

i know there's not much tooling for multiple patches worth of the DATA file, but it still has an effect.

Reply:
Acceleration uptime would be incorrect for older patches.  I didn't think about the from patch list.  I'll update it to 5.1 to 5.1